services/order.py

```diff
@@ -0,0 +1,43 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+
+from django.db import transaction
+
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime = None,
+) -> Order:
+    user = get_user_model().objects.get(username=username)
+    order = Order.objects.create(
+        user=user
+    )
+
+    if date:
+        order.created_at = date
+        order.save()

```

Comment:
Setting the `created_at` field manually might not be necessary, as it is typically set automatically by the `auto_now_add=True` attribute in the model. Consider whether this manual setting is needed or if it could lead to unintended behavior.